cantabular/static_dataset.go

```diff
@@ -41,49 +34,15 @@ type DimensionsTable struct {
 // loading the whole response to memory.
 // Use this method only if large query responses are NOT expected
 func (c *Client) StaticDatasetQuery(ctx context.Context, req StaticDatasetQueryRequest) (*StaticDatasetQuery, error) {
-	if c.gqlClient == nil {
-		return nil, dperrors.New(
-			errors.New("cantabular Extended API Client not configured"),
-			http.StatusServiceUnavailable,
-			nil,
-		)
-	}
-
-	logData := log.Data{
```

Comment:
Is it possible to add this log statement or something similar back in somewhere please?  They can be really useful when trying to debug.  Thanks.